lib/ansible/modules/cloud/vmware/vmware_serial_port.py

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Anusha Hegde <anushah@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: vmware_serial_port
+
+short_description: Create a serial port
+
+version_added: "1.0"
+
+description:
+    - "Creates a serial port on an existing VM"
+
+options:
+    hostname:
+        description:
+            - The host name or IP of the VCenter Server
+        required: true
+    username:
+        description:
+            - Username required to login to the VCenter Server
+        required: true
+    password:
+        description:
+            - Password required to login to the VCenter Server
+        required: true
+    name:
+        description:
+            - The name of the Virtual Machine
+        type: str
+    vm_id:
+        description:
+            - The id of the Virtual Machine
```

Comment:
Could you please add description on which ID to specify - `instance_uuid` or `BIOS_uuid` ? also add [`use_instance_uuid`](https://github.com/ansible/ansible/blob/a99af710fbfb768745b214f1a4de2f0dd5d40551/lib/ansible/modules/cloud/vmware/vmware_guest_boot_manager.py#L49) parameter to match other modules.